src/core/ledger/policies/immediate.js

```diff
@@ -22,16 +22,19 @@ export type ImmediatePolicy = {|
 
 /**
  * Split a grain budget in proportion to the cred scores in
- * the most recent time interval.
+ * the most recent time interval, with the option to extend the interval
+ * to include the last {history} weeks.
  */
 export function immediateReceipts(
   budget: G.Grain,
-  identities: ProcessedIdentities
+  identities: ProcessedIdentities,
+  history: number = 1
```

Comment:
confusing parameter name.

Reply:
Writing tests now.  For parameter name, how does `credIntervalCount` feel?  It's a bit wordy, but avoids the confusion around "credIntervals" which is used to describe an `IntervalSequence`.

Regarding the motivation, I thought this was a natural and useful generalization such that users could add a non-discounted "recent" policy (let's work out naming clashes).  For instance, I might want to reward contributors for the past 3 months, but not use the progressive discounting within that range.